app/src/main/res/layout/fragment_timeplay_action.xml

```diff
@@ -27,6 +28,71 @@
 
     <LinearLayout
         android:layout_width="match_parent"
+        android:layout_height="100dp"
+        android:background="@color/black"
+        android:orientation="horizontal"
+        android:visibility="@{viewModel.isSettingsPanelShown ? View.VISIBLE : View.GONE}"
+        >
+      <LinearLayout
+          android:layout_width="0dp"
+          android:layout_height="match_parent"
+          android:layout_weight="0.5"
+          android:orientation="vertical"
+          >
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="10dp"
```

Comment:
use one of the standard padding size in dimens xml
"spacing small/medium/..."

Reply:
done